docs/dictionary/control_st/return.lcdoc

```diff
@@ -2,39 +2,142 @@ Name: return
 
 Type: control structure
 
-Syntax: return <value> 
+Syntax: return [ value | error ] <value>
```

Comment:
Hmm... it says it passes the docs test and fails the IDE tests, however, I think the above should fail the docs test so I will investigate why it's not.